src/vehicle_move.cpp

```diff
@@ -564,7 +565,9 @@ void vehicle::thrust( int thd, int z )
         // helicopters improve efficiency the closer they get to 50-70 knots
         // then it drops off as they go over that.
         // see https://i.stack.imgur.com/0zIO7.jpg
-        if( is_rotorcraft() && is_flying_in_air() ) {
+        // rotorlike aircraft are intended to be mechanically similar to helicopters
+        // so they get the same efficiency factor
```

Comment:
.......why?

Again, this is a rotor physics issue. If you aren't using rotors, you should not be dealing with their physics.

Just to further clarify: As the rotor goes faster the relative speed of the airflow over its lifting surface *goes down*, which makes it produce less lift. Hence the "pushing power" at 20mph is not equivalent to the very same "pushing power" at 70mph.

You do not have a rotor. This should not apply to you.